### PR TITLE
Closes #179 R files generated by `create_sdtms_data.R` compliant with `styler()`

### LINKED ISSUE #179: Documentation:  Ensure the generated R files created by `create_sdtms_data.R` are compliant with `styler()`
### Please select a category the issue is focused on?

Function Documentation

### Let us know where something needs a refresh or put your idea here!

The R files generated by `create_sdtms_data.R` are not compliant with `styler()` when it contains the `xxTESTCD`/`xxTEST` table, so it’s necessary to run `styler()` on each file individually after generation.
Remove the space after the last cell, and add a space before the last `}`

### PR DESCRIPTION
**Pull request by Fanny-Gautier**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by Lina2689** [comment]
@manciniedoardo Can you please help us on why `create_sdtms_data.R` is taking datasets (the .rda files) from pharmaversesdtm package from CRAN? Should it not be taken from the [data](https://github.com/pharmaverse/pharmaversesdtm/tree/main/data) folder instead? As this is causing the undocumented field for variable labels every time you run until the updated datasets are submitted on CRAN. 

Please refer below line of code from `create_sdtms_data.R` 
"data(list = dataset_name, package = "pharmaversesdtm")"

@Fanny-Gautier Looks like library(metatools) is not necessary. You can remove it.

File: NEWS.md [hunk 1]
```diff
@@ -32,6 +32,7 @@
 -  Ensured all datasets have populated variables labels (#162)
 -  Changed units of insulin and triglycerides in `lb_metabolic` (#167)
 -  Fixed typo in `tu_onco_recist` dataset (#171)
+-  Ensured R files generated by `create_sdtms_data.R` are compliant with `styler()` (#179)
```
Comment (manciniedoardo) [inline NEWS.md:35]:
```suggestion
-  Ensured R files generated by `create_sdtms_data.R` are compliant with `{styler}` (#179)
```

**Comment by Lina2689** [comment]
> > @manciniedoardo Can you please help us on why `create_sdtms_data.R` is taking datasets (the .rda files) from pharmaversesdtm package from CRAN? Should it not be taken from the [data](https://github.com/pharmaverse/pharmaversesdtm/tree/main/data) folder instead? As this is causing the undocumented field for variable labels every time you run until the updated datasets are submitted on CRAN.
> > Please refer below line of code from `create_sdtms_data.R` "data(list = dataset_name, package = "pharmaversesdtm")"
> > @Fanny-Gautier Looks like library(metatools) is not necessary. You can remove it.
> 
> @Lina2689 I think you are using the CRAN-installed version of `{pharmaversesdtm}` rather than the local version that you can load using `devtools::load_all()`. Try the following (worked for me):
> 
> * Uninstall `{pharmaversesdtm}: `remove.packages("pharmaversesdtm")`
> * Reset RStudio session
> * Load the  local version of `{pharmaversesdtm}`: `devtools::load_all()` or Ctr+Shift+L
> * Run `create_sdtms_data.R`
> 
> let me know if this works! Otherwise all good except a tiny note in the `NEWS.md`.

Thanks @manciniedoardo Now it's good.

**Review by Lina2689 — APPROVED** [review]

**Review by manciniedoardo — APPROVED** [review]
Just made a quick update to NEWS.md myself so that we can go ahead and merge and close this one :)
